bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/digitalinvoice/help/view/DigitalInvoiceHelpNavigationBarBottomAdapter.kt

```diff
@@ -31,7 +32,7 @@ class DefaultDigitalInvoiceHelpNavigationBarBottomAdapter: DigitalInvoiceHelpNav
     var viewBinding: GbsHelpNavigationBarBottomBinding? = null
 
     override fun setOnBackButtonClickListener(listener: View.OnClickListener) {
-        viewBinding?.gbsGoBack?.setOnClickListener(listener)
+        viewBinding?.gbsGoBack?.setIntervalClickListener(listener)
```

Comment:
[Suggestion] Can't we pass in an instance of `IntervalClickListener`? It is an implementation of `View.OnClickListener` and we could prevent quick double tap issues also for custom adapters.

Instead of this:
`adapter.setOnBackButtonClickListener() { v: View? -> onBackPressed() }`

We could do this:
`adapter.setOnBackButtonClickListener( IntervalClickListener { v: View? -> onBackPressed() })`

Then we don't have to modify the default adapter implementations. If you think this could work, then it would be great if you could apply this change for all our view adapters.